Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -44,37 +47,36 @@ public enum RecognizerStartBehavior { AutoStart, ManualStart };
         public KeywordAndResponse[] KeywordsAndResponses;
 
         private KeywordRecognizer keywordRecognizer;
-        private Dictionary<string, UnityEvent> responses;
+        private List<string> responses = null;
```

Comment:
Intentionally left uninitialized.  Responses are set in the Inspector window of the Editor.